app/assets/stylesheets/reports.scss

```diff
@@ -1,3 +1,131 @@
 // Place all the styles related to the Reports controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import "colors";
+
+.reports-sidebar {
+  background: $sidebar-background;
+}
+
+.sidebar-title {
+  padding: 14px 0;
+  color: #767676;
+  font-size: 18px;
+
+}
+
+.sidebar-tabs {
+  .tabs {
+    display: flex !important;
```

Comment:
Try to avoid using !important whenever possible, it's better to just have more specific scoping